src/custom-element.js

```diff
@@ -1640,6 +1653,19 @@ function createBaseCustomElementClass(win) {
 
     /**
      * Returns an optional overflow element for this custom element.
+     * @return {!./layout-delay-meter.LayoutDelayMeter}
+     */
+    getLayoutDelayMeter_() {
+      if (!this.layoutDelayMeter_) {
+        /** @private {!./layout-delay-meter.LayoutDelayMeter} */
+        this.layoutDelayMeter_ = new LayoutDelayMeter(
```

Comment:
I think you should still declare it in the constructor as nullable and equal `null`. Otherwise, V8 may switch it to hash access.